### ISSUE TITLE
[Bug] Survival Bugs

### ISSUE THREAD

**Issue by Gallt** [tsunamayo/Starship-EVO#5636]
I meant to post this about 2 weeks ago after i wrote down 22 different bugs but work got wild and had to end up helping someone move in a way more active way than i thought I'd have to, but I'm getting around to this now. I've glossed over the updates and crossed out the ones that seemed to line up on my list, so I'll be posting the rest, but forgive me if it's something that has been posted already.

I verified each of these before posting but If you want video of any of these, let me know, should be able to re-create most of these:

1. When you are mining, if you hit "tab" and switch to inventory before the rocks reach you, you will be launched high into the air when they finally do.

2. When i drag an item to the trashcan icon on the right in the hotbar, it will delete everything in all the other hotbar slots in that row and the object which i dragged there is thrown to the ground.
![image](https://github.com/tsunamayo/Starship-EVO/assets/94942440/c26ec853-5cda-407b-8813-9e0092265303)

3. If i place a stone block that has color, and then right-click to remove it and get the block back, it always comes back as neutral grey to my inventory (it will stack with others that became grey in the same way regardless of what colors they wall were before)

4. There is a graphic bug that causes ore/ingots to stick to your hand no matter what you got equipped. I couldn't replicate this one and can't see if it was fixed already. but thought i'd bring it up
![image](https://github.com/tsunamayo/Starship-EVO/assets/94942440/798de024-e804-461f-a453-79ccc8efb76f)

5. "Refil all tanks" from the tab-menu entity breakdown works in survival

6. This one is sorta hard to follow: I had a base with a station block set to lock. I had a shipyard on a rail (so i could raise/lower what i was looking on). i built something that went further than the bounds and when i lowered it, it went underground. When you go underground with "No Clip (f5)" on, it pushes you out. When the ship went underground it lifted the entire base up out of the ground.
![image](https://github.com/tsunamayo/Starship-EVO/assets/94942440/92ccc791-be37-4327-b274-00d17e2c67a6)

Not sure if these are bugs, but will point them out.

1. Weight on rocks seems too high. i hadn't tested since survival began but 20 equaling about 2k lbs makes storage difficult considering we need 8 to make 1 silicon ingot. A re-balance of ore/stone weight i feel is in order, but it feels like an oversight to me.

2. Decals, in survival, when moved to the hotbar have a "0" under them, implying they need to be crafted (you cant use them). I'm not sure if you will have to manufacture them, but if not they are being treated like a crafted block.

3. Hover vehicles can't dock to shipyards (or have targeting). i could have swore they did once but i rarely touched hover vehicles.

The rest of the "22" i had listed are all fixed from how it seems. Again, if you have any questions, let me know, i should be able to record them. I have feedback on survival which will be another post when i have the time.

Keep it up!

**Comment by tsunamayo** [comment]
Hi, Issue 5) fixed.

**Comment by tsunamayo** [comment]
Decal issue fixed.

**Comment by tsunamayo** [comment]
Issue 3) is fixed.

**Comment by tsunamayo** [comment]
Issue 2)  I think you mistook it for a drop region, it is something that clear the entire row. Now all the item row are dropped in the world.
For 1) and 4) we have a case booked for it.
For 5) could you send me your world so I can understand how to reproduce? 
2-1 and 2-3 have been raised elsewhere.
Thanks for booking the bug, next time though dont hesitate to book a separate case for each one, as it is easier for me to track.

**Comment by Gallt** [comment]
Been trying for 45 min to zip my world file to 25mb or under and i cant get it smaller than 26-somthing -.- any other step you want me to take?
![image](https://github.com/tsunamayo/Starship-EVO/assets/94942440/5a5f11c2-ea8e-4a6f-9b35-35ff730df7cc)

**Comment by ProPeach** [comment]
@Gallt you could upload your word file to Google Drive/Dropbox/Mega/another file sharing site and paste the link here

**Comment by Gallt** [comment]
here is a link to the zip in my dropbox. In the future ill list bugs/feedback/suggestions in individual threads too for your ease of use. 
https://1drv.ms/u/s!AijOl7LKcLFiinysHkYcQnXd1n7C?e=MA8mSK

**Comment by tsunamayo** [comment]
@Gallt Hi, thanks it was helpful. Shipyard issue will be fixed for 24w11a. Be aware that it now the ship will traverse the ground instead of stopping the children entity (which would be the logical things to do), as the tech only allows this (joints are not simulated). At least it wont push the terrain entity upward. Thanks
